clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -3107,8 +3126,17 @@ private int processKafkaDataMessage(
           delete = ((Delete) leaderProducedRecordContext.getValueUnion());
         }
         keyLen = keyBytes.length;
-
+        // Commenting out, since only supporting value transformation for now deleting a key should be the same
+        // if (recordTransformer != null) {
```

Comment:
Let's just delete this for now and bring it back when come back to key transformation later.  We'll still have the history here in this PR for later if we ever need to copy it back.